sdks/python/apache_beam/io/gcp/datastore/v1/helper.py

```diff
@@ -49,6 +49,12 @@
 
 # pylint: enable=ungrouped-imports
 
+try:
```

Comment:
This one-liner has the same effect:

```python
from past.bulitins import cmp
``

Reply:
@superbobry A perfect suggestion.  Thanks much.  Done.